package/boot/grub2/grub2-efi/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=grub-efi
```

Comment:
A separate package could be avoided if build variant were supported for Host packages :\

So, i made a common.mk Makefile inspired from the toolchain/gcc to ease the maintainability of these grub packages. I will take a look on how you did that :)